src/scripts/plugin.js

```diff
@@ -158,8 +158,13 @@ const loadPlugin = () => {
       CKEDITOR.dialog.add('binderDialog', dialogConfig);
     },
   });
-
-  CKEDITOR.config.extraPlugins = 'enableBinder';
+  
+  //initialize if not already
+  if(!CKEDITOR.config.extraPlugins)
+    CKEDITOR.config.extraPlugins = '';
+  
+  CKEDITOR.config.extraPlugins += 'enableBinder';
+  config.toolbar[12].push('enableBinder');
```

Comment:
I am thinking that `plugin.js` should not contain any MindTouch specific code so that other people in the community will be about to use this plugin as well. I think moving it to `registerPlugin.js` is a better way to do.

Reply:
~~If you are moving it to registerPlugin, I think you may need a wrapper function since the config is a parameter/argument.~~
Looks like you took care of this in faad2e8